public/src/reimbursement-policy.md

```diff
@@ -2,107 +2,121 @@
   a[href]:after { content : "" }
 </style>
 
-Version: 1.0
-Updated: March 27, 2022
+Version: 1.1
+Updated: July 25, 2022
 
 # Speedcubing Canada Reimbursement Policy
```

Comment:
I assume this isn't supposed to be exhaustive. We should add something that allows someone to provide an explanation for an item not covered by this policy.

Reply:
I would prefer we explicitly add items, which is possible, but intentionally takes a bit of effort. An open ended option is too flexible in my opinion.

Are there specific items you think we're missing?

Reply:
I didn't really have anything in mind at the time. Reading through the NAC financial report we haven't explicitly included radios, dollies, number pads, or Square-1 tabs. They are covered by either tools, laptop accessories, or printing, but it may be worth adding them.

Re: open ended being too flexible, I'm okay with that if that's your intention.